wiki/Gameplay/Score/ScoreV1/osu!/en.md

```diff
@@ -1,31 +1,51 @@
----
-needs_cleanup: true
----
-
-# osu! scoring system
+# ScoreV1 (osu!)
 
 *See also: [osu! judgement system](/wiki/Gameplay/Judgement/osu!)*
 
-The score given by each hit circle and end of a slider is calculated with the following formula:-
+In **ScoreV1**, each of the object types in osu! is scored slightly differently.
+However, the total score is a simple sum of points awarded for each individual object in the beatmap.
+
+The rules for scoring each individual object type are outlined in the sections below.
+
+## Hit circles
+
+Each hit circle is assigned a numerical point value using the formula below:
+
+`Score = Hit value * ((Combo multiplier * Difficulty multiplier * Mod multiplier) / 25)`
+
+where:
+
+- The *hit value* is the numerical value of the hit circle judgement (50, 100, or 300).
+- The *combo multiplier* is equal to (combo before this hit - 1) or 0, whichever is higher.
```

Comment:
so with a combo of 0 or 1 (start of beatmap or after each combo reset) you get 0 points for the next object?

Reply:
No, this is just wrong. Clerical error when touching the old (correct) formula. Fixed in https://github.com/ppy/osu-wiki/pull/7525/commits/d2f28f879d4a17ee154bf25ca3c586d6f38a8621.